.gitignore

```diff
@@ -24,5 +24,4 @@ res/
 etc/state-migration-test/target/
 
 node_modules/*
-artifacts/
```

Comment:
Why remove this?

Reply:
Its a duplicate, just a nit that I noticed.